tests/bare/server.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+import flwr as fl
+
+
+# Start Flower server
+hist = fl.server.start_server(
+    server_address="0.0.0.0:8080",
+    config=fl.server.ServerConfig(num_rounds=3),
+)
+
+if hist.losses_distributed[-1][1] != 0:
+    with open("result", "w") as file:
+        file.write("FAIL")
+    sys.exit("Loss did not decrease.")
+else:
+    with open("result", "w") as file:
+        file.write("SUCCESS")
+    sys.exit()
```

Comment:
Can we just set a 0 or 1 exit code here and use that as the exit code of `test.sh`? That would prevent us from having to write a file etc.